tests/services/conftest.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from fastapi_azure_auth.user import User
+
+
+def build_user():
```

Comment:
would be good to name this something different to that in the other conftest but not essential.